### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by hugok79** [rNeomy/auto-tab-discard#367]
Hi there,
The translations are coming from Transifex or via Pull Request?
Thanks in advanced.

**Comment by rNeomy** [comment]
Hi, Transifex is better. You can ping me here for merge when it is ready.

**Comment by hugok79** [comment]
Ok, thanks.
However, I have a question. In Transifex I see the **Portuguese** `pt` team and the **Portuguese (Portugal)** `pt_PT` team, which I have now also asked to join now. Since `pt` and `pt_PT` are the same language, I'd like to know which one I should translate. Although **pt** is the language code for Portuguese, the **pt_PT** language code was also created so that there would be no mixing between Portuguese from Portugal and Portuguese from Brazil. Portuguese is the official language of Brazil, but due to the increasingly obvious differences between the Portuguese spoken in Brazil and the official Portuguese (the one spoken in Portugal), the **pt_PT** language code has been more widely adopted so that there is no mixing.

**Comment by hugok79** [comment]
Another thing, I only see the pt_BR locale in the code:
https://github.com/rNeomy/auto-tab-discard/tree/master/v2
https://github.com/rNeomy/auto-tab-discard/tree/master/v3

**Comment by hugok79** [comment]
The ´Portuguese` translation was recently completed by a translation by a colleague of mine in Transifex

**Comment by rNeomy** [comment]
Right, I just removed the `pt_PT` from Transifex. Also the `pt` language is now available on the latest release. I just forgot to sync the git repo.

https://github.com/rNeomy/auto-tab-discard/tree/master/v3/_locales

**Comment by hugok79** [comment]
@rNeomy Ok, thank you.
Also thanks for this very useful extension
